revisions/J0/src/data/diacritics.asm

```diff
@@ -0,0 +1,20 @@
+; The Japanese text uses two diacritics:
+; - 1: ゛
+; - 2: ゜
+DiacriticsTable::
+    db     0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0,   0
```

Comment:
Maybe make the table a bit more compact with less whitespace, like in the other revisions?